lib/fetch/fetchRequest.ts

```diff
@@ -72,7 +72,7 @@ function fetchRequest(method: string, url: string, args: FetchOptions) {
         return formatResult(status, data);
       })
       .then(result => {
-        if (error) {
+        if (error || result.responseJSON?.error) {
```

Comment:
how about throwing different errors for 200 and not 200, so it can provide meaningful debugging information for both devs and our support team.

Reply:
it looks like error object already includes information about request [status](https://github.com/okta/okta-auth-js/blob/d018c1b2d997b303dba553e0f7947515aa3087e6/lib/fetch/fetchRequest.ts#L36)